README.md

```diff
@@ -1,46 +1,48 @@
 # .NET SDK for Lucidtech AI Services
 
+## Documentation
+
+[Link to docs](https://docs.lucidtech.ai/dotnet/v1/index.html)
+
 ## Installation
 
 ### NuGet
 
-```
-$ nuget install Lucidtech.LAS.Receipt
+```bash
+$ nuget install Lucidtech.LAS
 ```
 
 ## Usage
-Instantiate the Lucidtech.LAS.Client with your API key:
 
-``` C#
-using Lucidtech.LAS;
+### Preconditions
 
-// ...
+- Documents must be in upright position
+- Only one receipt or invoice per document is supported
+- Supported file formats are: jpeg, pdf
 
-string apiKey = "...";
-var client = new Client(apiKey);
-```
+### Quick start
 
-To scan a receipt, call the API either with a URL to an image file:
 ```C#
-string receiptUrl = "...";
-List<Detection> detections = client.ScanReceipt(receiptUrl);
+using Lucidtech.Las;
+
+ApiClient apiClient = new ApiClient("<api endpoint>");
+Prediction response = apiClient.Predict(documentPath: "document.pdf",modelName: "invoice");
```

Comment:
Missing whitespace before "modelName".
Can we add an example of dumping the results to standard out?